Automattic-Tracks-iOS/TestHelpers.swift

```diff
@@ -0,0 +1,54 @@
+import Foundation
+import ObjectiveC
+import Sentry
+
+typealias EventLoggingCallback = (Event) -> ()
+typealias DecisionCallback = (Bool) -> ()
+
+private var errorLoggingCallback: EventLoggingCallback? = nil
+private var messageLoggingCallback: EventLoggingCallback? = nil
+private var eventSendCallback: DecisionCallback? = nil
+
+internal extension CrashLoggingDataProvider {
+
+    var didLogErrorCallback: EventLoggingCallback? {
```

Comment:
I'm not completely happy with this design, tbh. I think I'd like to rework it with a `CrashLoggingDelegate` protocol, but I don't want to expose Sentry the `Event`, since that's an implementation detail. 

For now, because it's really only used as a hook system for testing (and because it's an `internal` extension, the compiler will be able to prove this is dead code and leave it out of builds) I'm ok to leave it, but I think as we add more tests, this'll need a little bit more polish :)

Reply:
> I'm ok to leave it, but I think as we add more tests, this'll need a little bit more polish :)

Sounds good to me. 👍